cookbooks/galera/recipes/server.rb

```diff
@@ -184,11 +189,24 @@
 end
 
 my_ip = node['ipaddress']
-init_host = galera_config['init_node']
+
+if node[:galera_config]["use_hostnames"] then
```

Comment:
You should use the galleria_config hash from before, because if node[:galera_config] doesn't exist, this line will fail.